core/domain/suggestion_registry.py

```diff
@@ -2001,3 +2001,511 @@ def __init__(
+    """Dictionary representing the TranslationSubmitterTotalContributionStats
+    object for frontend.
+    """
+
+    language_code: str
+    contributor_id: str
+    topic_ids_with_translation_submissions: List[str]
+    recent_review_outcomes: List[str]
+    recent_performance: int
+    overall_accuracy: float
+    submitted_translations_count: int
+    submitted_translation_word_count: int
+    accepted_translations_count: int
+    accepted_translations_without_reviewer_edits_count: int
+    accepted_translation_word_count: int
+    rejected_translations_count: int
+    rejected_translation_word_count: int
+    first_contribution_date: str
+    last_contributed_in_days: int
+
+
+class TranslationSubmitterTotalContributionStats:
+    """Domain object for the TranslationSubmitterTotalContributionStatsModel."""
+
+    def __init__(
+        self,
+        language_code: str,
+        contributor_id: str,
+        topic_ids_with_translation_submissions: List[str],
+        recent_review_outcomes: List[str],
+        recent_performance: int,
+        overall_accuracy: float,
+        submitted_translations_count: int,
+        submitted_translation_word_count: int,
+        accepted_translations_count: int,
+        accepted_translations_without_reviewer_edits_count: int,
+        accepted_translation_word_count: int,
+        rejected_translations_count: int,
+        rejected_translation_word_count: int,
+        first_contribution_date: datetime.date,
+        last_contribution_date: datetime.date
+    ) -> None:
+        self.language_code = language_code
+        self.contributor_id = contributor_id
+        self.topic_ids_with_translation_submissions = (
+            topic_ids_with_translation_submissions)
+        self.recent_review_outcomes = recent_review_outcomes
+        self.recent_performance = recent_performance
+        self.overall_accuracy = overall_accuracy
+        self.submitted_translations_count = submitted_translations_count
+        self.submitted_translation_word_count = submitted_translation_word_count
+        self.accepted_translations_count = accepted_translations_count
+        self.accepted_translations_without_reviewer_edits_count = (
+            accepted_translations_without_reviewer_edits_count
+        )
+        self.accepted_translation_word_count = accepted_translation_word_count
+        self.rejected_translations_count = rejected_translations_count
+        self.rejected_translation_word_count = rejected_translation_word_count
+        self.first_contribution_date = first_contribution_date
+        self.last_contribution_date = last_contribution_date
+
+    def to_dict(self) -> TranslationSubmitterTotalContributionStatsDict:
+        """Returns a dict representation of a
+        TranslationSubmitterTotalContributionStats domain object.
+
+        Returns:
+            dict. A dict representation of a
+            TranslationSubmitterTotalContributionStats domain object.
```

Comment:
Nit: For such a simple method like this and since we already mentioned the return value in the docstring summary, we can probably simplify the return description to eliminate redundancy, e.g.

```suggestion
            dict. The dict representation.
```

Ditto below.

Reply:
Sounds reasonable. Changed.